sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/async_client_base.py

```diff
@@ -104,3 +134,27 @@ def _build_pipeline(config: Configuration, transport: AsyncHttpTransport, **kwar
     @property
     def vault_url(self) -> str:
         return self._vault_url
+
+    @property
+    def _models(self):
```

Comment:
This imports a module every time it's called. The import will be faster the second time but it would be better to import only once, considering the wanted module won't change. Also, `_create_client` could handle importing the models.